### DISCUSSION TITLE
Mqtt connection

### DISCUSSION THREAD

**Discussion by mircotamburini** [unoplatform/uno#8828] [Q&A]
I would like to connect to Mqtt broker with UWP and WASM application how can i do it?
my mqtt broker have  tcp (1883) classic connection and also websocket port opened

**Comment by jeromelaban** [comment]
Uno does not provide anything out of the box for those scenarios, though if you want to target Wasm, you'll need to use WebSockets for this. We haven't tried it, but https://github.com/dotnet/MQTTnet would look like a good fit.
